torch/functional.py

```diff
@@ -981,6 +984,114 @@ def cdist(x1, x2, p=2., compute_mode='use_mm_for_euclid_dist_if_necessary'):
     else:
         raise ValueError("{} is not a valid value for compute_mode".format(compute_mode))
 
+def atleast_1d(*tensors):
```

Comment:
Do you really need to document this here? Seems like it could go in _torch_docs.py?

Reply:
Tried moving them to `_torch_docs` but on importing got

```
E   TypeError: don't know how to add docstring to type 'function'
```

I guess keeping the documentation in `functional.py` is in-spirit with the functions already present in that file.

Reply:
Hm. You should definitely be able to add them to _torch_docs since these are new docstrings! 

Docstrings should really be in _torch_docs.py. Having them elsewhere is a pain. I don't think you need to define these functions in that file at all.

Reply:
I agree that having the else is not intuitive to find ( faced that while trying to find `meshgrid`) but I think this is where it will have to stay here.

>  I don't think you need to define these functions in that file at all.

The reason the function is in `functional.py` is so that it can support the `torch.atleast_1d(x,y,z)` signature similar to numpy which does. The signatures in `native_functions.yaml` are wrapped to support the above signature.
```
- func: atleast_1d(Tensor self) -> Tensor
  use_c10_dispatcher: full
  variants: function

- func: atleast_1d.Sequence(Tensor[] tensors) -> Tensor[]
  use_c10_dispatcher: full
```

I tried this 
```python
>>> torch._C._add_docstr(torch.atleast_1d, "Test Doc")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: don't know how to add docstring to type 'function'
>>> torch._C._add_docstr(torch.gt, "Test Doc")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
RuntimeError: function 'gt' already has a docstring
```

Also I looked at `Module.cpp` which has the code for `_add_docstr`.
https://github.com/pytorch/pytorch/blob/c20426f86d2b06f93812a38ea513ee3803900ab6/torch/csrc/Module.cpp#L210
the type of the new `atleast_1d` function falls in the following error branch.
https://github.com/pytorch/pytorch/blob/c20426f86d2b06f93812a38ea513ee3803900ab6/torch/csrc/Module.cpp#L260

Also checking the type of few functions, the type of the functions added in `functional.py` is different from the ones which are present in `_torch_docs.py`.
```python
>>> torch.atleast_1d # functional.py
<function atleast_1d at 0x7f4c3336c510>
>>> torch.meshgrid # functional.py
<function meshgrid at 0x7f4c3329a950>
>>> torch.add # _torch_docs
<built-in method add of type object at 0x7f4c5c45f000>
>>> torch.combinations # _torch_docs
<built-in method combinations of type object at 0x7f4c5c45f000>
```